src/components/VariationsMainView/VariationsMainView.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { MuiThemeProvider, createMuiTheme } from '@material-ui/core/styles';
+import blue from '@material-ui/core/colors/blue';
+
+import BubbleDisplay from '../BubbleDisplay/BubbleDisplay';
+import SingleBubble from '../SingleBubble/SingleBubble';
+import AudioTransportBar from '../AudioTransportBar/AudioTransportBar';
+import ZoomControls from '../ZoomControls/ZoomControls';
+import TimelineMetadata from '../TimeMetadata/TimeMetadata';
+
+import {
+  AppBar,
+  Toolbar,
+  Typography,
+  TextField,
+  IconButton,
+} from '@material-ui/core';
+import {
+  Info,
+  AddCircle,
+  Delete,
+  CloudDownload,
+  Settings,
+} from '@material-ui/icons';
+
+const TempAppBar = props => (
```

Comment:
Think we need a ticket for the app bar?

Reply:
Created the ticket #70, refactored the component